sdks/builds/llvm.mk

```diff
@@ -1,9 +1,16 @@
 
 LLVM_SRC?=$(TOP)/sdks/builds/toolchains/llvm
 
-$(TOP)/sdks/builds/toolchains/llvm:
```

Comment:
Why don't we just keep the existing code since it does the same thing?

Reply:
The code in versions.mk reads the hashes from SUBMODULES.json and makes sure that the proper version is checked out if the hash changes. Its also used by acceptance-tests/ so there is code sharing.